src/cfront.c

```diff
@@ -1514,24 +1514,35 @@ void read_body_statement(block_t *parent)
 
     if (lex_accept(T_while)) {
         ir_instr_t *false_jump;
-        ir_instr_t *start = add_instr(OP_label); /* start to return to */
-
+        ir_instr_t *start_jump =
```

Comment:
Can you confirm the case `for (;;) break;` as well?

Reply:
should I send another PR? or also support `for (;;) break;` in this PR?

Reply:
Let's consolidate the "break" support for both `while` and `for` loop in this pull request.

Reply:
Now break is valid in `do_while`, `for` and `while` !

Reply:
good job